### DISCUSSION TITLE
When I use ''blank' in the Link tag and 'prefetch' is set to 'true', does the prefetching still take effect?

### DISCUSSION THREAD

**Discussion by little077** [vercel/next.js#67895] [Help]
### Summary

When I use ''blank' in the Link tag and 'prefetch' is set to 'true', does the prefetching still take effect?

**Comment by icyJoseph** [comment]
Well, looking at the current implementation, it only checks for `prefetch` prop 🤔 not the `_blank`. 

- https://github.com/vercel/next.js/blob/canary/packages/next/src/client/link.tsx

I took a quick look, didn't bother to test it out...

You might still want to prefetch though, in case it is an ISR page, not yet generated, the `prefetch`, will trigger it to be created, and available for the user when they click on it.
